bears/go/GoTypeBear.py

```diff
@@ -0,0 +1,19 @@
+import re
+
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+
+
+class GoTypeBear(LocalBear, Lint):
+    executable = 'gotype'
+    arguments = "-e {filename}"
```

Comment:
> -e
	report all errors (not just the first 10)
https://godoc.org/golang.org/x/tools/cmd/gotype